routes/coachRoutes.js

```diff
@@ -0,0 +1,7 @@
+const express = require('express');
+const router = express.Router();
+const coachController = require('../controllers/coachController');
+
+router.post('/initial-survey', coachController.addInitialSurvey);
```

Comment:
Can you make the initial survey url for client and coach different? like maybe "/initial-coach-survey" and "initial-client-survey"

Reply:
done